client/footer.ejs

```diff
@@ -1,5 +1,12 @@
-<footer flex="none">
-  <a href="/about">About</a>
-  <a href="/tos">Terms</a>
-  <a href="/privacy-policy">Privacy</a>
+<footer layout="row">
+  <span flex class="footer-left">
+    <a href="/about">About</a>
+    <a href="/tos">Terms</a>
+    <a href="/privacy-policy">Privacy</a>
+  </span>
+  <span flex></span>
+  <span flex class="footer-right">
+    <a href="https://github.com/porygonco/porybox">GitHub</a> |
```

Comment:
I can know what GitHub is while still not immediately realizing why a site is randomly linking to it. The link isn't important because it happens to go to GitHub, it's important because it contains our source code.